pymongo/command_cursor.py

```diff
@@ -301,6 +301,9 @@ def _try_next(self, get_more_allowed):
         else:
             return None
 
+    def try_next(self):
```

Comment:
Let's add a docstring here similar to ChangeStream.try_next. We should also mention the new cursor_command() and try_next() methods in the changelog.

Reply:
Got it, added both.